temperature/crud.py

```diff
@@ -0,0 +1,65 @@
+import datetime
+
+from fastapi import HTTPException, Depends, status
+from fastapi.responses import JSONResponse
+from sqlalchemy.orm import Session
+from sqlalchemy.future import select
+from sqlalchemy.ext.asyncio import AsyncSession
+from sqlalchemy.exc import IntegrityError
+from websockets.legacy import async_timeout
```

Comment:
The import from `websockets.legacy` is not used in the code. Consider removing it if it's not needed to keep the code clean and maintainable.